webrtc.html

```diff
@@ -8263,8 +8263,11 @@ <h3><dfn>RTCIceTransport</dfn> Interface</h3>
       </ol>
 
       <p>When the <a>ICE Agent</a> indicates that it finished gathering a
-      <a>generation</a> of candidates for an <code><a>RTCIceTransport</a></code>, the
-      user agent MUST queue a task that runs the following steps:</p>
+      <a>generation</a> of candidates for an <code><a>RTCIceTransport</a></code>,
+      the user agent MUST wait until all candidates it has indicated as
```

Comment:
Rather than "must wait", why not say "when the ICE agent is finished gathering, and all candidates for that ICETransport have been surfaced to the application, the UA MUST queue a task...."